src/Rbac.php

```diff
@@ -22,20 +25,18 @@ class Rbac extends AbstractIterator
     protected $createMissingRoles = false;
 
     /**
-     * @param  bool                     $createMissingRoles
-     * @return \Zend\Permissions\Rbac\Rbac
+     * @param  bool $createMissingRoles
+     * @return void
      */
-    public function setCreateMissingRoles($createMissingRoles)
+    public function setCreateMissingRoles(bool $createMissingRoles): void
```

Comment:
For consistency with other packages, and until we finish up zend-coding-standard, please add a space on either side of the `:` when specifying the return type hint: `) : void`. This applies to all RTH added in this patch.

Reply:
Done

Reply:
@weierophinney  I know it's not my place to speak but I don't think there should be a space on the left side of the colon. All php.net code snippets (e.g. [here](http://php.net/manual/en/migration71.new-features.php)) have it without it and many IDE's (such as PhpStorm) default to not having it too.

Having a space to the left would mean extra configuration would need to be specified only to achieve something that's contrary to the current "standard".